### PR TITLE
Add IDFFMode do FBP PS IOC

### PR DESCRIPTION
**Pull request by xresende**
- passed initial tests with IOC for TB correctors
- running for all PS IOCs

### CODE REVIEW

File: siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py [hunk 1]
```diff
@@ -339,6 +346,20 @@ def scope_read(self, device_id):
...
             return _dcopy(psupply.scope)
 
+    # --- IDFF parameters ---
+
+    def idff_mode_set(self, state):
+        """Change IDFF mode: True or False."""
+        self._idff_mode = state
+        if state:
+            while not self._queue.empty():  # wait until queue is empty
+                pass
```
Comment (anacso17) [inline siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py:356]: maybe a little sleep here?

Comment (xresende) [reply siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py]: done

File: siriuspy/siriuspy/cycle/conn.py [hunk 2]
```diff
@@ -619,11 +620,24 @@ def check_sofbmode(self, state, wait=1):
         state = _PSConst.OffOn.On if state == 'on' else _PSConst.OffOn.Off
         return _pv_timed_get(self['SOFBMode-Sts'], state, wait=wait)
 
+    def set_idffmode(self, state):
+        """Set IDFFMode."""
+        state = _PSConst.OffOn.On if state == 'on' else _PSConst.OffOn.Off
```
Comment (murilobalves) [inline siriuspy/siriuspy/cycle/conn.py:625]: maybe use state.lower() == 'on' just to also allow 'ON' and 'OFF'?

**Review by murilobalves — APPROVED** [review]

File: siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py (hunk 1 above)
Comment (xresende) [reply siriuspy/siriuspy/pwrsupply/pructrl/prucontroller.py]: @anacso17 , this PR is basically tested already.

**Review by anacso17 — APPROVED** [review]

File: siriuspy/siriuspy/cycle/conn.py (hunk 2 above)
Comment (xresende) [reply siriuspy/siriuspy/cycle/conn.py]: done

**Pull request merged**